src/components/DisplayArticles.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import { getArticles } from "../modules/articles";
+import { Card, Image } from "semantic-ui-react";
+
+class DisplayArticles extends Component {
```

Comment:
Have you tried to use a functional component with `useState` and `useEffect` hook?